cmd/wit-bindgen-go/cmd/generate/generate.go

```diff
@@ -140,25 +142,28 @@ func parseFlags(cmd *cli.Command) (*config, error) {
 		cmd.Bool("versioned"),
 		cmd.Bool("force-wit"),
 		path,
+		cmd.Bool("verbose"),
```

Comment:
Where is this flag defined in the `Command`?

Reply:
It's not -- turns out `verbose` is a built in -- if you try to define it, it complains about it already being defined! 

See also https://github.com/bytecodealliance/wasm-tools-go/pull/214#discussion_r1820023324

Reply:
I have a branch that implements this. I had to make some changes (-v is an alias for --version).

Will push later today.

Reply:
Done. See #220.

I had to make some related changes to wire in a leveled logger through other packages.

I also added -vv as an alias for --debug to support debug logging.

Reply:
Hey that branch looks great! Should I close this PR and we'll go with #220  ?

Reply:
Rebase! Let's keep your test.

Reply:
Done! Thank you for hopping on this so quickly -- I'll make the target of this PR your branch :)